attack-theme/templates/macros/matrix.html

```diff
@@ -89,6 +88,7 @@ <h3 class="text-center matrix-title">{{matrix.name}}</h3>
     </div>
     {% if not isIndex %}
     <div class="timestamp deemphasis text-center pt-3">Last modified: {{matrix.timestamp}}</div>
+    {% set permalink_title = "the matrices" if matrix.tactics | length > 1 else "the matrix" %}
```

Comment:
I think this is dead code from an earlier implementation, correct?

Reply:
I don't think that it is dead code. It's used for the tooltip of the Version Permalink button.

Reply:
`permalink_title` only occurs once in the matrix macro: https://github.com/mitre-attack/attack-website/blob/feature/%23191-add-versioning-to-tour/attack-theme/templates/macros/matrix.html#L91